src/adv.py

```diff
@@ -49,3 +59,42 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+while True:
+    # * Prints the current room name
+    print(f'\nCurrent Location: {player.current_room.name[0]}')
+
+    # * Prints the current description (the textwrap module might be useful here).
+    print(f'\nLocation Description: {player.current_room.description}')
+
+    # commands
+    print('''
+        Explore the map by entering one of the following: 
+        "n" for North
+        "s" for South
+        "w" for West
+        "e" for East
+        ''')
+
+    print('Enter "q" to quit the game.')
+
+    # * Waits for user input and decides what to do.
+    current_room = player.current_room.name
+    choice = input('What would you like to do?')
+    if len(choice) != 1:
```

Comment:
You may want to change this later so that you can parse longer commands like "help" or "take". So this might look like if len(choice) == 0: